sql/core/src/main/scala/org/apache/spark/sql/execution/HiveResult.scala

```diff
@@ -65,6 +65,11 @@ object HiveResult {
     // database, table name, isTemp.
     case command @ ExecutedCommandExec(s: ShowTablesCommand) if !s.isExtended =>
       command.executeCollect().map(_.getString(1))
+    // SHOW TABLE EXTENDED in Hive do not have isTemp while our v1 command outputs isTemp.
```

Comment:
it's not only about `isTemp`. Now we only display the `information` column, not even the table name?